src/java.base/share/classes/java/util/zip/Inflater.java

```diff
@@ -46,7 +46,7 @@
  * This class inflates sequences of ZLIB compressed bytes. The input byte
  * sequence is provided in either byte array or byte buffer, via one of the
  * {@code setInput()} methods. The output byte sequence is written to the
- * output byte array or byte buffer passed to the {@code deflate()} methods.
+ * output byte array or byte buffer passed to the {@code inflate()} methods.
```

Comment:
This looks okay but probably should be a link as it's first use of inflate.

Reply:
I had thought about it too, but I realized that it talks of inflate "methods" (there are more than one inflate methods with different parameters). So I suspect it was intentional to use `{@code}` instead of `{@link}`.

Reply:
Okay, we can leave it as a code tag. At some point I suspect the class description of both Inflater and Deflater will need to be replaced as they are missing so much.